### DISCUSSION TITLE
How to Use This Lib?

### DISCUSSION THREAD

**Discussion by KiddoV** [dream-num/univer#92] [Q&A | 常见问题] [answered]
How do I use this lib, I don't see any package on ``npm``.

**Comment by jikkai** [comment]
Thanks for your interest in the project!
We are continuing our efforts to ensure project quality, so univer has not been published to npm yet. 😢

**Reply by KiddoV** [reply]
Thanks for reply. Do you guys have ETA when we will see the alpha version for it?

**Reply by jikkai** [reply, marked as answer]
Perhaps in December of this year, we will release an early preview version 🤘

**Reply by KiddoV** [reply]
Nice! Looking forward to use the lib!
